.github/workflows/check_format.yml

```
@@ -53,7 +53,7 @@ jobs:
 
       - name: Format sources
         env:
-          FINDENT_FLAGS: "-Rr -i2 -d3 -f3 -s3 -w3 -t3 -j3 -k5 --ws_remred --indent_ampersand --openmp=0"
+          FINDENT_FLAGS: "-Rr -i2 -d3 -f3 -w3 -t3 -j3 -k5 --ws_remred --indent_ampersand --openmp=0"
```

This is fixed in a separate PR.
```suggestion
          FINDENT_FLAGS: "-Rr -i2 -d3 -f3 -s3 -w3 -t3 -j3 -k5 --ws_remred --indent_ampersand --openmp=0"
```